.circleci/config.yml

```diff
@@ -172,18 +172,17 @@ node-plugin-base: &node-plugin-base
           - .nyc_merge/*
 
 node-upstream-base: &node-upstream-base
+  parameters:
+    node-version:
+      type: string
+      default: "12"
   docker:
-    - image: node:12
+    - image: node:<< parameters.node-version >>
   working_directory: ~/dd-trace-js
   resource_class: small
   steps:
-    - checkout
-    - *yarn-versions
-    - *restore-yarn-cache
-    - run:
-        name: Install dependencies
-        command: yarn install
-    - *save-yarn-cache
+    - checkout-and-yarn-install:
+        yarn-args: ""
```

Comment:
Why is this needed?

Reply:
The base case includes `--ignore-scripts`. That's why it typically wasn't reused, and instead we'd end up with stuff like this all over the place: https://github.com/DataDog/dd-trace-js/pull/1505/files/a5eb8c09fed6c7fc466160819eff96958e9d7396#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L185

Reply:
That link doesn't work for me, but what's the problem with just keeping the `--ignore-scripts` everywhere?

Reply:
🤷‍♂️ I dunno. We didn't previously have it everywhere. instead we put manual `yarn install` commands in everywhere. I just kept the same effect with this.

Reply:
Ok, it's fine to leave it like that then, but I think it wasn't on purpose that there was multiple ways to call `yarn install`.